lib/bom.js

```diff
@@ -368,6 +368,11 @@ declare class CloseEvent extends Event {
     wasClean: boolean;
 }
 
+type MessageEventHandler = (event: MessageEvent) => mixed;
+type CloseEventHandler = (event: CloseEvent) => mixed;
+type MessageEventListener = {handleEvent: MessageEventHandler} | MessageEventHandler;
```

Comment:
Indeed, running a random test (the `fetch` test) from the Flow test suite now takes >14 seconds rather than ~0.8 seconds. That's very unfortunate.